core/templates/pages/topic-editor-page/modal-templates/create-new-subtopic-modal.template.html

```diff
@@ -10,16 +10,17 @@ <h3>
     </div>
     <input class="form-control protractor-test-new-subtopic-title-field" style="font-size: 12px;"
            placeholder="Enter a brief description for the subtopic that students can easily understand, e.g. 'Adding Fractions'"
-           ng-model="$ctrl.subtopicTitle" ng-change="$ctrl.resetErrorMsg()" maxlength="<[MAX_CHARS_IN_SUBTOPIC_TITLE]>" autofocus>
+           ng-model="$ctrl.subtopicTitle" ng-change="$ctrl.resetErrorMsg()" maxlength="<[$ctrl.MAX_CHARS_IN_SUBTOPIC_TITLE]>" autofocus>
     <span class="oppia-input-box-subtitle">
       <i>
-        Subtopic title should be at most <[MAX_CHARS_IN_SUBTOPIC_TITLE]> characters.
+        Subtopic title should be at most <[$ctrl.MAX_CHARS_IN_SUBTOPIC_TITLE]> characters.
```

Comment:
Thanks @Showtim3! Can you also make this change in `subtopic-editor-tab.component.html`?

Reply:
OK I just checked, this issue no longer exists in that file. Resolving comment.

Reply:
Hey @kevintab95 I verified the subtopic-editor-tab.component.html and it looks fine to me? Can you point out which line?